docs/ru/getting-started/example-datasets/nyc-taxi.md

```diff
@@ -6,16 +6,16 @@ sidebar_label: "Данные о такси в Нью-Йорке"
 
 # Данные о такси в Нью-Йорке {#dannye-o-taksi-v-niu-iorke}
 
-Этот датасет может быть получен двумя способами:
+Этот массив может быть получен двумя способами:
```

Comment:
This makes it worse, because the meaning of "массив" is ambiguous: while it could be interpreted as a dataset, it could also mean "an array", which is a data type in ClickHouse.

Reply:
«Массив» was in previous title. «Набор данных»?